app/helpers/application_helper.rb

```diff
@@ -18,6 +18,15 @@ def multiline_text(string)
     ApplicationController.helpers.sanitize(string.gsub("\n", '<br>'), tags: %w[br])
   end
 
+  def further_information_needed
+    if current_application.further_informations.empty?
+      false
+    else
+      last_further_info = current_application.further_informations.order(:created_at).last.created_at
+      (current_application.status == 'sent_back') && (last_further_info >= current_application.app_store_updated_at)
```

Comment:
Handily the `enum` magic means (I learned recently) we can do `current_application.sent_back?` instead of `current_application.status == 'sent_back'`